rebuild/src/addon.h

```diff
@@ -26,28 +26,35 @@ using std::endl;
     try                                      \
     {
 
-#define WORKER_TRY_CATCH_END(name)                              \
-    }                                                           \
-    catch (Napi::Error & err)                                   \
-    {                                                           \
-        SetError(err.Message());                                \
-        goto out_err;                                           \
-    }                                                           \
-    catch (std::exception & err)                                \
-    {                                                           \
-        std::ostringstream msg;                                 \
-        msg << "caught exception in " #name ": " << err.what(); \
-        SetError(msg.str());                                    \
-        goto out_err;                                           \
-    }                                                           \
-    catch (...)                                                 \
-    {                                                           \
-        SetError("caught unknown exception in " #name);         \
-        goto out_err;                                           \
-    }                                                           \
-                                                                \
-    out_err:                                                    \
-    ThrowJsException();                                         \
+#define WORKER_TRY_CATCH_END(name)                                 \
```

Comment:
I see this defined everywhere, can we abstract out to a single header file which can be included everywhere